.azure-pipelines/integration-tests.yml

```diff
@@ -444,7 +444,7 @@ jobs:
       projects: |
         test/test-applications/integrations/**/*.csproj
         !test/test-applications/integrations/dependency-libs/**/*.csproj
-      arguments: --configuration $(buildConfiguration) -p:Platform=$(buildPlatform) -p:ManagedProfilerOutputDirectory=$(publishOutput) -p:BuildInParallel=false /nowarn:netsdk1138
+      arguments: --configuration $(buildConfiguration) -p:Platform=$(buildPlatform) -p:ManagedProfilerOutputDirectory=$(publishOutput) /nowarn:netsdk1138 -p:ExcludeManagedProfiler=true -p:ExcludeNativeProfiler=true -p:LoadManagedProfilerFromProfilerDirectory=false
```

Comment:
Can you please keep `-p:BuildInParallel=false` here as well?

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1171/commits/e0b070b25e6ea9301cf9dbed9f9b15b6a52855ed